packages/office-ui-fabric-react/src/components/Slider/Slider.test.tsx

```diff
@@ -99,14 +99,6 @@ describe('Slider', () => {
     expect(component.find('.ms-Slider-slideBox').prop('aria-valuetext')).toEqual(values[selected]);
   });
 
-  it('formats the value when a format function is passed', () => {
-    const value = 10;
-    const valueFormat = (val: any) => `${val}%`;
-    const component = mount(<Slider value={value} min={0} max={100} showValue={true} valueFormat={valueFormat} />);
-
-    expect(component.find('label.ms-Label.ms-Slider-value').text()).toEqual(valueFormat(value));
```

Comment:
Regarding my comment of removing the label from the DOM is even breaking a test that was put in place so it definitely raises a red flag.

Reply:
should I readd this test cause now ive re added showvalue

Reply:
yes